windows/setup_py2exe.py

```diff
@@ -0,0 +1,296 @@
+logger.info('Checking for GTK')
+assert_exist(GTK_DIR)
+
+logger.info('Checking PyGTK+ library')
+assert_module('pygtk')
+
+logger.info('Checking Python win32 library')
+assert_module('win32file')
+
+logger.info('Checking for UPX')
+assert_exist(UPX_EXE, 'Download UPX executable from http://upx.sourceforge.net/')
+
+logger.info('Checking for CodeSign.bat')
+check_exist('CodeSign.bat', 'Code signing is not available')
+
+logger.info('Checking for NSIS')
+check_exist(NSIS_EXE, 'NSIS executable not found: will try to build portable BleachBit')
+
+
+#if not fast:
+#    logger.info('echo Pre-compressing executables')
+#    python_install_path =  os.path.dirname( sys.executable )
+#
+#    python_files = recursive_glob(python_install_path, ['*.pyd'])
+#    for file in python_files:
+#        compress(UPX_EXE, UPX_OPTS, file)
+#
+#    gtk_files = recursive_glob(GTK_DIR, ['*.exe', '*.dll'])
+#    for file in gtk_files:
+#        compress(UPX_EXE, UPX_OPTS, file)
+#
+#    pywintypes27 = python_install_path + '\\Lib\\site-packages\\pywin32_system32\\pywintypes27.dll'
+#    compress(UPX_EXE, UPX_OPTS, pywintypes27)
+
+logger.info('Deleting directories build and dist')
+shutil.rmtree('build', ignore_errors=True)
+shutil.rmtree('dist', ignore_errors=True)
+
+logger.info('Running py2exe')
+shutil.copyfile( 'bleachbit.py', 'bleachbit_console.py')
+cmd= sys.executable +  ' -OO setup.py py2exe'
+run_cmd(cmd)
+assert_exist('dist\\bleachbit.exe')
+assert_exist('dist\\bleachbit_console.exe')
+os.remove('bleachbit_console.py')
+
+
+if not os.path.exists('dist'):
+    os.makedirs('dist')
+
+logger.info('Copying GTK files and icon')
+shutil.copyfile( GTK_DIR + '\\bin\\intl.dll',  'dist\\intl.dll')
+
+shutil.copytree(GTK_DIR + '\\etc', 'dist\\etc')
+shutil.copytree(GTK_DIR + '\\lib', 'dist\\lib')
+shutil.copytree(GTK_DIR + '\\share', 'dist\\share')
+shutil.copyfile( 'bleachbit.png',  'dist\\share\\bleachbit.png')
+
+
+#logger.info('Compressing executables')
+#files = recursive_glob('dist', ['*.pyd', '*.dll', '*.exe'])
+#for file in files:
+#    compress(UPX_EXE, UPX_OPTS, file)
+
+
+logger.info('Purging unnecessary GTK+ files')
+cmd= sys.executable +  ' setup.py clean-dist'
+run_cmd(cmd)
+
+
+logger.info('Copying BleachBit localizations')
+shutil.rmtree('dist\\share\\locale', ignore_errors=True)
+shutil.copytree('locale', 'dist\\share\\locale')
+assert_exist( 'dist\\share\\locale\\es\\LC_MESSAGES\\bleachbit.mo' )
+
+logger.info('Copying BleachBit cleaners')
+if not os.path.exists( 'dist\\share\\cleaners'):
+    os.makedirs( 'dist\\share\\cleaners')
+cleaners_files = recursive_glob('cleaners', ['*.xml'])
+for file in cleaners_files:
+    shutil.copy( file,  'dist\\share\\cleaners')
+
+
+logger.info('Checking for CleanerML')
+assert_exist( 'dist\\share\\cleaners\\internet_explorer.xml' )
+
+
+logger.info('Checking for Linux-only cleaners')
+if os.path.exists( 'dist\\share\\cleaners\\wine.xml'):
+    logger.info('grep -l os=.linux. dist/share/cleaners/*xml | xargs rm -f')
```

Comment:
Rather than just logging the grep command, would you please actually delete all files in `dist/share/cleaners/*xml` that match the pattern `os="linux"`?